test/core/end2end/end2end_tests.h

```diff
@@ -39,12 +39,17 @@
 typedef struct grpc_end2end_test_fixture grpc_end2end_test_fixture;
 typedef struct grpc_end2end_test_config grpc_end2end_test_config;
 
+/* Test feature flags. If FEATURE_MASK_SUPPORTS_X is included in the
+ * feature_mask of a grpc_end2end_test_config then tests of feature X will be
+ * disabled for that test fixture. */
 #define FEATURE_MASK_SUPPORTS_DELAYED_CONNECTION 1
 #define FEATURE_MASK_SUPPORTS_HOSTNAME_VERIFICATION 2
 #define FEATURE_MASK_SUPPORTS_PER_CALL_CREDENTIALS 4
 #define FEATURE_MASK_SUPPORTS_REQUEST_PROXYING 8
 #define FEATURE_MASK_SUPPORTS_CLIENT_CHANNEL 16
 #define FEATURE_MASK_SUPPORTS_AUTHORITY_HEADER 32
+#define FEATURE_MASK_SUPPORTS_RESOURCE_QUOTA_SERVER 64
```

Comment:
I think you want something like FEATURE_MASK_FORBIDS_XXX?
I think some if not all other flags works the other way and for easier integration with existing configs, you may want something to default to 0 (support) and set to 1 in places where it is not supported.